automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/previewmodal/PreviewSpec.scala

```diff
@@ -79,7 +79,10 @@ class PreviewSpec extends FreeSpec with WebBrowserSpec with WorkspaceFixtures wi
           //file sometimes changes but is always a JSON array, so easy test...
           filePreview should startWith("[")
           filePreview should endWith("]")
-          previewModal.getPreviewMessage shouldBe "Previews may not be supported for some filetypes."}
+          previewModal.getPreviewMessage shouldBe "Previews may not be supported for some filetypes."
+
+          previewModal.xOut()
```

Comment:
I thought we didn't need this in our tests any more.

Reply:
yes, really annoying to see them in test log. you can find them in above Jenkin job log.